packages/init/package.json

```diff
@@ -7,7 +7,10 @@
   "exports": {
     ".": "./index.js",
     "./debug.js": "./debug.js",
-    "./pre.js": "./pre.js",
+    "./pre.js": {
+      "node": "./src/pre-node.js",
```

Comment:
This is a fine exception to the rule I mention, because it only applies in cases where `exports` are respected.